### DISCUSSION TITLE
Angular 11 support

### DISCUSSION THREAD

**Discussion by droshev** [SAP/fundamental-ngx#4052] [Ideas]
Let's start a discussion about Angular 11 support in fundamental-ngx library

**Comment by droshev** [comment]
Our estimation for migration ngx-core to Angular 11/TypeScript 4 is couple of days.
Is there something else we should consider?

**Reply by fkolar** [reply]
Today's migration should be pretty smooths. I guess. we can open up discussion on the next status meeting or review.
